### ISSUE TITLE
Project "Lualize"

### ISSUE THREAD

**Issue by SDXC** [fmd-project-team/FMD#37]
One of my main targets since I forked this project is to port all Pascal website modules to Lua modules.

Saying that: @AbidZakaria are you interested to port some of these modules to Lua? Please let me know and if you are interested I will post the instructions on how we will doing that.

**Comment by MIRVOUS** [comment]
so, can you post the instructions.
or it is not for the public

**Comment by AbidZakaria** [comment]
I'm interested

**Comment by Tmp341** [comment]
``8muses`` seems an adult site, but ``translate.webtoons.com`` seems like it is a multi-language translation project.

**Comment by SDXC** [comment]
@AbidZakaria I will build a special version of fmd.exe where all these Pascal modules are excluded. So you can just start creating Lua scripts. You don't actually have to understand Pascal to get hints on how the algorithms work. So you can still check the Pascal modules as a reference:
https://github.com/fmd-project-team/FMD/tree/master/baseunits/modules

Please don't commit/pull request these new lua scripts. Zip them and attach them to this issue. I will create a branch later and commit it there. You will be able to make the pull requests in the new branch afterwards.

**Comment by SDXC** [comment]
@AbidZakaria x64 is okay with you, right?

[fmd_0.9.161.0_Win64-withoutPascalModules.zip](https://github.com/fmd-project-team/FMD/files/3050784/fmd_0.9.161.0_Win64-withoutPascalModules.zip)

Please don't forget to use this version separately.

**Comment by AbidZakaria** [comment]
@SDXC is okay

**Comment by anfgomezmo** [comment]
I will start with some Lua modules, if I have any advance, I will let you know

**Comment by AbidZakaria** [comment]
Manga Webtoons are in progress and I'm trying on an issue https://github.com/fmd-project-team/FMD/issues/57

**Comment by SDXC** [comment]
Webtoons is now implemented. Will be released in the next few minutes/hours.

**Comment by AbidZakaria** [comment]
@SDXC LeoManga done moved from pascal to lua file 
and fixed from issue https://github.com/fmd-project-team/FMD/issues/60

[LeoManga.zip](https://github.com/fmd-project-team/FMD/files/3141612/LeoManga.zip)

**Comment by AbidZakaria** [comment]
@SDXC and @Tmp341 please remove [NewType] site sorce
because that site not manga reader

**Comment by SDXC** [comment]
@AbidZakaria What do you mean? It is a reading website.

**Comment by AbidZakaria** [comment]
can you try go to that website for testing

**Comment by SDXC** [comment]
FMD doesn't support the new website design right now, but there are titles you can read there (not all chapters are complete though (preview or premium content, not sure).

For example:
https://comic.webnewtype.com/contents/sewayaki/
https://comic.webnewtype.com/contents/sewayaki/1340/

**Comment by SDXC** [comment]
If you can't access the website, then I'll look into fixing this.

**Comment by SDXC** [comment]
@AbidZakaria Could you port and fix this website? https://github.com/fmd-project-team/FMD/issues/54

**Comment by SDXC** [comment]
@AbidZakaria Here's a new version without Pascal modules. cf_bypass is also included.
[fmd_0.9.163.1_Win64-withoutPascalModules.zip](https://github.com/fmd-project-team/FMD/files/3145544/fmd_0.9.163.1_Win64-withoutPascalModules.zip)

**Comment by Tmp341** [comment]
Wouldn't it be good to create branches for these?

**Comment by SDXC** [comment]
That's not necessary because none of these ports are in conflict with the master branch.
Lua modules are higher priority than Pascal modules, so even if we release a Lua module before a version without the Pascal module comes out, it still works.

New branches will only be created if we do something outside the normal release cycle.

**Comment by Tmp341** [comment]
But we do need a new database repository.

**Comment by SDXC** [comment]
Yep, will be new repo. At the moment I'm more focused on fixing and adding lua modules and the cf_bypass. A bit later, I'll start with the new repo for website db files.

**Comment by SDXC** [comment]
Modules to port with higher priority:
- [ ] Batoto / MangaWindow
- [ ] HentaiCafe
- [ ] MangaInn

**Comment by dazedcat19** [comment]
Please continue this project. I'd like to release a new version after completing this project. Most of the file in modules seems to be very simple. Those bellow 10kb can be easily ported with a template.
@Slasar41 can you look at it

**Comment by Slasar41** [comment]
Sorry I can't help.
After seeing the remaining Pascal modules, it's looks complex for me.
I only know the basic like xpath not json, regex, encrypt, etc.
